components/java/scripts/start.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
```

Comment:
can we call this `validate.py`?

Ideally, I would like the validations to take place AFTER `start` in each node, but since there's no way to add a validate lifecycle operation, `start` is a good place. In the case of `java` and `python`, there IS no start, so creating a `validate.py` script makes more sense.